src/main/java/mate/academy/model/User.java

```diff
@@ -1,8 +1,8 @@
 package mate.academy.model;
 
 public class User {
-    private String email;
-    private String password;
+    private final String email;
+    private final String password;
```

Comment:
Don`t change it

Reply:
Yes, 
![image](https://github.com/mate-academy/jv-oop-lesson-0/assets/63095184/50b557b7-3fc9-4e4b-8f4c-c1860700979f)